lib/src/model/class.dart

```diff
@@ -201,6 +201,16 @@ class Class extends Container
   @override
   String get fileName => "${name}-class.html";
 
+  @override
+  String get filePath {
+    if (!identical(canonicalModelElement, this)) {
```

Comment:
I basically copied `href`'s implementation in all these files and removed the `${package.baseHref}`, but I'm not sure this is the right approach.